pkg/generated/restapi/configure_fulcio_server.go

```diff
@@ -105,12 +105,16 @@ func configureAPI(api *operations.FulcioServerAPI) http.Handler {
 	// fulcio: Generic PKCS11 / HSM backed servic
 	api.SigningCertHandler = operations.SigningCertHandlerFunc(pkgapi.SigningCertHandler)
 
+	api.HomepageHandler = operations.HomepageHandlerFunc(pkgapi.HomePageHandler)
+
 	api.PreServerShutdown = func() {}
 
 	api.ServerShutdown = func() {}
 
 	api.AddMiddlewareFor("POST", "/api/v1/signingCert", middleware.NoCache)
 
+	api.AddMiddlewareFor("GET", "/", middleware.NoCache)
```

Comment:
instead of `middleware.NoCache`, how about just writing the bytes for the HTML here:
```
api.AddMiddlewareFor("GET", "/", func(w http.ResponseWriter, req *http.Request) {
    fmt.Fprintf(w, homepageBytes)
  })
```

Reply:
thanks for your Review and time Bob, i will address the comments!